### ISSUE TITLE
Rails 7 raw_filter method removed

### ISSUE THREAD

**Issue by nickhammond** [zdennis/activerecord-import#736]
It looks like Rails 7 removes the `raw_filter` method from `ActiveSupport::Callbacks::Callback` which triggers an undefined method error when calling `import!`.

```
undefined method `raw_filter' for #<ActiveSupport::Callbacks::Callback:0x00007fedffe98e38>
```

It looks like in 6.1 `raw_filter` was just reading the `@filter` variable so maybe just changing the call to `filter` instead would be fine for this. My use case is pretty simple and I ended up just swapping this out for `insert_all!` and I noticed the other discussions around future Rails support as well.

```
def raw_filter; @filter; end
```

https://github.com/rails/rails/blob/6-1-stable/activesupport/lib/active_support/callbacks.rb#L305

https://github.com/zdennis/activerecord-import/blob/82ac6f9c90b216be4f1125b36ba7d0d1608dfa6c/lib/activerecord-import/import.rb#L37

**Comment by jkowens** [comment]
Hey thanks for the heads up!

**Comment by jkwuc89** [comment]
Q: When will a release of this gem that contains this fix be available at https://rubygems.org/gems/activerecord-import?

**Comment by ZilvinasKucinskas** [comment]
Releasing new version of the gem would be great! 🚀  

I can verify that my application test suite passes with `activerecord-import` master branch:

```ruby
gem 'activerecord-import', git: "https://github.com/zdennis/activerecord-import"
```

**Comment by jkowens** [comment]
The fix has been released in v1.3.0 🎉
